tests/manage/monitoring/prometheus/test_capacity.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs import constants
+from ocs_ci.utility import prometheus
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-899")
+def test_capacity_workload_rbd_alerts(
+    workload_storageutilization_80p_rbd,
+    workload_storageutilization_90p_rbd
```

Comment:
I would probably organized the tests vice versa, grouping the 90p alerts for cephfs and rbd into single test case. But this works as well.